core/subsurface-qt/DiveObjectHelper.cpp

```diff
@@ -403,13 +403,29 @@ QString DiveObjectHelper::getCylinder() const
 
 QString DiveObjectHelper::startPressure() const
 {
-	QString startPressure = getPressures(m_dive, START_PRESSURE);
+	QString startPressure;
+	for (int i = 0; i < MAX_CYLINDERS; i++) {
+		if (is_cylinder_used(m_dive, i)) {
+			if (startPressure.isEmpty())
+				startPressure = getPressures(m_dive, i, START_PRESSURE);
+			else
+				startPressure = startPressure + ", " + getPressures(m_dive, i, START_PRESSURE);
```

Comment:
Depending on your locale you have a comma as decimal separator this might mess things up.
![img_4376](https://user-images.githubusercontent.com/30494/42389273-6f9cddbc-8148-11e8-943b-e662099dd9fb.jpg)

For some reason we are showing 1 decimal digit when editing.

Reply:
excellent catch. We should separate groups with `;` for that reason.

Reply:
Or we remove the decimal precision, frankly there is no point in showing anything other than full integers.

Reply:
That is one option, but it just need to be consistent. The desktop shows with 1 digit.

If you then edit on mobile and round it, effectively the user data is being changed without warning (even though it's <1bar).